js/stores/appStore.js

```diff
@@ -115,7 +115,7 @@ const createWindow = (browserOpts, defaults) => {
     // Neither a frame nor a titlebar
     // frame: false,
     // A frame but no title bar and windows buttons in titlebar 10.10 OSX and up only?
-    titleBarStyle: 'hidden',
+    titleBarStyle: 'hidden-inset',
```

Comment:
This has been changed in Electron, `hidden-inset` will be deprecated with 2.0, use `hiddenInset` instead: [native_window_mac.mm](https://github.com/electron/electron/blob/6aef8f4fea5a019e8f9c4a6205c042b3fbc98dff/atom/browser/native_window_mac.mm#L424)